samples/type-ahead-search-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,78 @@
+---
+page_type: sample
+description: Demonstrating the feature of type ahead search (static and dynamic) control on Adaptive Cards.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "24-12-2021 23:30:17"
+---
+
+# Type ahead search control on Adaptive Cards
```

Comment:
Please incorporate comments from C# sample to this solution as well.

https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/216#discussion_r774894460
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/216#discussion_r774895992